src/generic/data/apiHooks.test.js

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import { useOrganizationListData } from './apiHooks';
+
+jest.mock('@tanstack/react-query', () => ({
+  useQuery: jest.fn(),
+}));
+
+jest.mock('./api', () => ({
+  updateContentTaxonomyTags: jest.fn(),
+}));
+
+describe('useOrganizationListData', () => {
+  it('should return success response', () => {
+    const orgs = ['org1', 'org2'];
+    useQuery.mockReturnValueOnce({ isSuccess: true, data: orgs });
+    const result = useOrganizationListData();
+
+    expect(result).toEqual({ isSuccess: true, data: orgs });
+  });
+
+  it('should return failure response', () => {
+    useQuery.mockReturnValueOnce({ isSuccess: false });
+    const result = useOrganizationListData();
+
+    expect(result).toEqual({ isSuccess: false });
+  });
+});
```

Comment:
I don't think this test is doing anything. 

`useOrganizationListData` is only calling a single function and returning its result, and that is the very function being mocked. So other than potentially adding coverage, this is doing nothing. 

I think it's better to just delete this file and test it with the component.

Reply:
Makes sense, I removed it and removed the mocking from the `useOrganizationListData` when testing the component, and mocked the underlying API instead so its tested properly. https://github.com/openedx/frontend-app-course-authoring/pull/755/commits/d922834de261158a0a9c722ddbcd4ff28a72a892